form/unit_testing.rst

```diff
@@ -157,40 +157,49 @@ before creating the parent form using the ``PreloadedExtension`` class::
     be getting errors that are not related to the form you are currently
     testing but to its children.
 
+Forms Using Validation
```

Comment:
@pierredup Can you remove this section? We can then still merge your fixes to the class namespaces which is still a legit fix.